ui/temboardui/static/src/views/Configuration.vue

```diff
@@ -19,58 +22,134 @@ const resetParamName = ref("");
 const resetParamValue = ref("");
 
 onMounted(() => {
-  if (props.error_code != 0) {
-    showError(props.error_message);
-  }
+  getCategories();
 });
 
-function generatePopoverContent(row) {
-  let content = `<table><tr><td>Type:</td><td><b>${row["vartype"]}</b></td></tr>`;
-  if (row["unit"]) {
-    content += `<tr><td>Unit:</td><td><b>${row["unit"]}</b></td></tr>`;
+function getCategories() {
+  $.ajax({
+    url: "/proxy/" + props.address + "/" + props.port + "/pgconf/configuration/categories",
```

Comment:
The template string notation would probably fit here and in a few other locations.